state/factory/workingset.go

```diff
@@ -526,11 +530,13 @@ func (ws *workingSet) processWithCorrectOrder(ctx context.Context, actions []*ac
 			return errors.Wrap(err, "error when run action")
 		}
 		receipts = append(receipts, receipt)
-		blkCtx.GasLimit -= receipt.GasConsumed
-		if fCtx.EnableDynamicFeeTx && receipt.PriorityFee() != nil {
-			(&blkCtx.AccumulatedTips).Add(&blkCtx.AccumulatedTips, receipt.PriorityFee())
+		if !action.IsSystemAction(act) {
```

Comment:
after discussion, it does not hurt to add this check, to make sure the behavior is the same as`pickAndRunActions()`